Packs/Humio/Integrations/Humio/Humio.yml

```diff
@@ -0,0 +1,356 @@
+- display: Humio URL
+  name: url
+  defaultvalue: https://cloud.humio.com
+  type: 0
+  required: true
+- display: User API token
+  name: API-key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Query to use to fetch incidents
+  name: queryParameter
+  defaultvalue: ""
+  type: 0
+  required: false
+  additionalinfo: Write the query to use to fetch incidents (only used for fetch-incidents)
+- display: Fetch incidents from repository
+  name: queryRepository
+  defaultvalue: ""
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: Fetch incidents from
+  name: queryStartTime
+  defaultvalue: 24h
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: TimeZoneOffset in Minutes
+  name: queryTimeZoneOffsetMinutes
+  defaultvalue: "0"
+  type: 0
+  required: false
+  additionalinfo: (only used for fetch-incidents)
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: humio-query
+    arguments:
+    - name: repository
+      required: true
+      description: Repository to search
+    - name: queryString
+      required: true
+      description: Query string to use
+    - name: start
+      description: Relative or absolute (epoch)
+      defaultValue: 24h
+    - name: end
+      description: Relative or absolute (epoch)
+      defaultValue: now
+    - name: isLive
+      description: Answer with true, 1, t, y or yes
+      defaultValue: "false"
+    - name: timeZoneOffsetMinutes
+      description: TimeZoneOffset in Minutes (default 0)
+      defaultValue: "0"
+    - name: arguments
+      description: Additional arguments
+    outputs:
+    - contextPath: Humio.Query
+      description: Query output
+      type: Unknown
+    description: Query the data from Humio
+  - name: humio-query-job
+    arguments:
+    - name: queryString
+      required: true
+      description: Query string to use
+    - name: start
+      description: Relative or absolute (epoch)
+      defaultValue: 24hours
+    - name: end
+      description: Relative or absolute (epoch)
+      defaultValue: now
+    - name: repository
+      required: true
+      description: Repository to use
+    - name: isLive
+      description: Is it live?
+      defaultValue: "False"
+    - name: timeZoneOffsetMinutes
+      description: Timezone offset in Minutes
+      defaultValue: "0"
+    - name: arguments
+      description: Additional Arguments
+    outputs:
+    - contextPath: Humio.Job
+      description: Query Job outputs
+      type: Unknown
+    description: Issue a query job to Humio
+  - name: humio-poll
```

Comment:
It seems that query is async action, and poll used to check if the query is done. If this is the case, then you should create a playbook for running and query and polling it's results until the query is over.
https://xsoar.pan.dev/docs/playbooks/generic-polling